src/pages/PageTransferDial/index.tsx

```diff
@@ -10,75 +11,86 @@ import { ScrollView, View } from 'react-native'
 import callStore from 'stores/callStore'
 import contactStore from 'stores/contactStore'
 import Nav from 'stores/Nav'
-import CustomColors from 'utils/CustomColors'
 
 @observer
 class PageTransferDial extends React.Component {
   prevId?: string
+
   componentDidMount() {
     this.componentDidUpdate()
```

Comment:
What is the need here? We should not call any lifecycle method directly. React handles these calls.

Reply:
This is existing code Jigar, I haven't written this part. I will remove it